library/time-library/src/test/java/uk/gov/gchq/gaffer/time/RBMBackedTimestampSetTest.java

```diff
@@ -111,6 +111,26 @@ public void testGetNumberOfTimestamps() {
         assertEquals(4, numberOfTimestamps);
     }
 
```

Comment:
Can you add a test for null values in start and end date, as well as for when start date is after end date?